### PR TITLE
Support different locations

### PR DESCRIPTION
**Pull request by ashcrow**
This is a proposed enhancement for #2 though it may also cover #3.

/cc @lucab @Rahuls0720

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,9 +59,10 @@ import (
 )
 
 func RequireFedora() error {
-	// The OSRelease instance
-	or, err := osrelease.New()
-
+	// The OSRelease instance using the default paths
+	or, err := osrelease.New(nil)
+	// Or to inspect the files in /sysroot
+	//or, err := osrelease.New(osrelease.PrefixDefaultFilePaths("/sysroot"))
```
Comment (lucab) [inline README.md]: Probably just a matter of taste, but I was think of leaving `New` as-is and having an additional constructor like `osrelease.WithOverrides(sysroot string, usrMountpoint string)`.

Comment (ashcrow) [reply README.md]: That works for me. I'm fine with reworking this to make a secondary constructor. @Rahuls0720 WDYT?

Comment (Rahuls0720) [reply README.md]: I agree with @lucab

Comment (ashcrow) [reply README.md]: Updated

**Pull request merged**